### PR TITLE
ATO-864: create dynamoDb infrastructure

### PR DESCRIPTION
**Pull request by isaac-GDS**
Create the infrastructure to be able to get store in dynamoDb

To review - setup config and naming.

### CODE REVIEW

File: ipv-stub-template.yml [hunk 1]
```diff
@@ -33,13 +33,30 @@ Globals:
       - !Ref AWS::NoValue
 
 Resources:
+  DynamoDB:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      TableName: FormResponse
+      KeySchema:
+        - AttributeName: TestId
```
Comment (billrdunn) [inline ipv-stub-template.yml:41]: Would `FormId` be a more suitable name?

**Comment by billrdunn** [comment]
Does it deploy to dev ok?

File: ipv-stub-template.yml (hunk 1 above)
Comment (isaac-GDS) [reply ipv-stub-template.yml:41]: This is a placeholder - as im not sure what the structure would be.  DynamoDb require this on setup

Comment (billrdunn) [reply ipv-stub-template.yml:41]: Oh actually we're going to store the form response against auth code so maybe the hash key should be `AuthCode`

**Review by billrdunn — APPROVED** [review]

File: ipv-stub-template.yml (hunk 1 above)
Comment (isaac-GDS) [reply ipv-stub-template.yml]: Updated to `FormId`

**Pull request merged**